targets/ipq40xx-generic

```diff
@@ -43,6 +43,17 @@ device('avm-fritz-repeater-1200', 'avm_fritzrepeater-1200', {
 })
 
 
+-- EnGenius
+
+device('engenius-ens620ext', 'engenius_ens620ext', {
+	factory = '-squashfs-factory_30',
+	factory_ext = '.bin',
+	extra_images = {
+		{'-squashfs-factory_35', '-factory_35', '.bin'},
+	},
```

Comment:
In tis special case, we should put both factory image with their indication as an extra image.

Reply:
Thought it would be nice to offer a "factory" image, but I can see that this will be confusing to some degree. Changed it as suggested.